test cases/unit/51 introspect buildoptions/meson.build

```diff
@@ -1,3 +1,13 @@
 project('introspect buildargs', ['c'], default_options: ['c_std=c11', 'cpp_std=c++14', 'buildtype=release'])
 
 subA = subproject('projectA')
+python_mod = import('python')
+python_ins = python_mod.find_installation('python3')
+
+r = run_command(python_ins.path(), 'c_compiler.py')
```

Comment:
Please use find_program('c_compiler.py') here instead of explicitly passing the script to the python interpreter.

Reply:
Sure. Maybe it should be also explicitly documented in `run_command` that `find_program` is preferred for scripts and that the interpreter should not be specified.